### PR TITLE
Suppress UI-elements if no potrace available

### PR DESCRIPTION
**Pull request by jpirnay**

### CODE REVIEW

File: meerk40t/gui/propertypanels/imageproperty.py [hunk 1]
```diff
@@ -1,8 +1,10 @@
 # import threading
 import wx
+from PIL import Image
```
Comment (tatarize) [inline meerk40t/gui/propertypanels/imageproperty.py:3]: I think this might harden the requirement for Pillow.

Comment (tatarize) [reply meerk40t/gui/propertypanels/imageproperty.py:3]: Nm. It's part of gui.

File: meerk40t/core/element_treeops.py [hunk 2]
```diff
@@ -1450,6 +1450,7 @@ def wlist_minus(node, **kwargs):
                 property_op(self.kernel.root, node)
             self.signal("element_property_update", [node])
 
+    @tree_conditional(lambda node: has_vectorize(node))
```
[the remark this answers is not in the document]
Comment (tatarize) [reply meerk40t/core/element_treeops.py:1453]: the lamba node: could just get called with has_vectorize just as simply.

File: meerk40t/gui/propertypanels/imageproperty.py [hunk 3]
```diff
@@ -956,7 +995,10 @@ def generate_preview(self):
 
     @staticmethod
     def accepts(node):
-        if node.type == "elem image":
+        # Changing the staticmethod into a regular method will cause a crash, so therefore this circumvention
+        # Not the nicest thing in the world, as we need to instantiate the class once to reset the status flag
+        global HAS_VECTOR_ENGINE
```
Comment (tatarize) [inline meerk40t/gui/propertypanels/imageproperty.py:1000]: This is only needed if we're going to set that value. It's unneeded if we're just checking its value.

File: meerk40t/core/element_treeops.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (jpirnay) [reply meerk40t/core/element_treeops.py:1453]: Agree, lazy me just copied and pasted a single line...

**Pull request merged**